string uppaalModel
string modelConfig
string adapterConfig
StateMachine stateMachine
